### PR TITLE
Fixed a mischaracterization of composite export.

### PR DESCRIPTION
**Pull request by parthaa**
Cherry-pick into:

* [x] Foreman 2.5 (Satellite 6.10)
* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

File: guides/doc-Content_Management_Guide/topics/Using_ISS.adoc [hunk 1]
```diff
@@ -100,7 +100,7 @@ This option is available for all content-export operations. You can use the `des
 === Exporting a Content View Version
 
 You can export a version of a Content View to an archive file from {ProjectServer} and use this archive file to create the same Content View version on another {ProjectServer} or on another {ProjectServer} organization.
-{Project} does not export composite Content Views.
+{Project} exports composite Content Views as a normal content view. The composite nature is not retained.
```
Comment (maximiliankolb) [inline guides/doc-Content_Management_Guide/topics/Using_ISS.adoc:103]:
```suggestion
{Project} exports composite Content Views as normal content views.
The composite nature is not retained.
```

**Review by melcorr — APPROVED** [review]
Thanks LGTM

File: guides/doc-Content_Management_Guide/topics/Using_ISS.adoc [hunk 2]
```diff
@@ -100,7 +100,8 @@ This option is available for all content-export operations. You can use the `des
 === Exporting a Content View Version
 
 You can export a version of a Content View to an archive file from {ProjectServer} and use this archive file to create the same Content View version on another {ProjectServer} or on another {ProjectServer} organization.
-{Project} does not export composite Content Views.
+{Project} exports composite Content Views as normal content views.
+The composite nature is not retained.
```
Comment (parthaa) [inline guides/doc-Content_Management_Guide/topics/Using_ISS.adoc:104]:
```suggestion
The composite nature is not retained.
On importing the exported archive, a normal content-view will get created/updated in the downstream {ProjectServer}.
```

File: guides/doc-Content_Management_Guide/topics/Using_ISS.adoc [hunk 3]
```diff
@@ -100,7 +100,9 @@ This option is available for all content-export operations. You can use the `des
...
+On importing the exported archive, a normal content-view will get created/updated in the downstream {ProjectServer}.
```
Comment (parthaa) [inline guides/doc-Content_Management_Guide/topics/Using_ISS.adoc:105]: @melcorr you ok with this ^ ?

Comment (melcorr) [reply guides/doc-Content_Management_Guide/topics/Using_ISS.adoc:105]: The word "normal" seems strange.

Comment (melcorr) [reply guides/doc-Content_Management_Guide/topics/Using_ISS.adoc:105]: Oh damn I edited your comment rather than quote @parthaa - I'm an idiot.

**Comment by melcorr** [comment]
Thanks @parthaa - if you're happy, I can merge :)

**Comment by parthaa** [comment]
> Thanks @parthaa - if you're happy, I can merge :)

Sounds good. There are more changes coming to this page. But I am ok doing it piece meal.

**Comment by lzap** [comment]
2.5 CP: 72f476b759ee449b2a67a324f3a61bd26f3fbac9

**Comment by lzap** [comment]
@ryandeussing hopefully I cherry picked all your requests, please do not comment on comments but rather continue discussion in pull requests. These are easier to track.

**Comment by lzap** [comment]
I thought this PR was not cherry picked, I was wrong. Sorry for the confusion, ignore my notes.

**Pull request merged**